src/PayID/pay-id-admin-client.ts

```diff
@@ -0,0 +1,78 @@
+import PayIDError from './pay-id-error'
+
+/* eslint-disable @typescript-eslint/require-await */
+// TODO(keefertaylor): Enable lint rules when method is implemented.
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable max-classes-per-file */
+
+/**
+ * An admin for Xpring's PayID service.
+ *
+ * Admin clients can create, update, and delete address mappings. To resolve PayIDs to addresses, please use `PayIDClient`.
+ *
+ * Inputs to this class are always in X-Addresses.
+ * @see https://xrpaddress.info/
+ *
+ * @warning This class is experimental and should not be used in production applications.
+ * TODO(keefertaylor): Export this class in index.ts when it's ready for external consumption.
+ */
+export default class PayIDAdminClient {
+  private static putEndpoint = '/v1/users'
+
+  /**
+   * Initialize a new PayIDAdminClient.
+   *
+   * TODO(keefertaylor): Add a remoteURL configuration property when it is used, otherwise the linter is upset about an unused property.
+   *
+   * @param authorizationToken An authorization token.
+   * @param remoteURL The remote URL for the PayID Service.
+   */
+  public constructor(
+    private readonly authorizationToken: string,
+    private readonly serviceURL: string,
```

Comment:
`remoteURL` in the docstring vs `serviceURL` here?

Reply:
Done